sdk/appcenter-push/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
This is really bad behavior of formatter. I don't know how to fix this one.

Reply:
We don't fix those, formatter wins, always.

Reply:
Agree that we default formatter wins. Sigh.